server/src/main/java/org/elasticsearch/search/aggregations/bucket/significant/SignificantTextAggregationBuilder.java

```diff
@@ -123,6 +123,22 @@ public AggregationBuilder parse(String aggregationName, XContentParser parser)
         };
     }
 
+    protected SignificantTextAggregationBuilder(SignificantTextAggregationBuilder clone) {
+        super(clone);
+        this.bucketCountThresholds = clone.bucketCountThresholds;
```

Comment:
It seems that BucketCountThresholds is mutable so I think we need to clone it as well?